resources/arduino_files/infineon_board/infineon_board.ino

```diff
@@ -0,0 +1,362 @@
+#include <stdlib.h>
+
+#include <OneWire.h>
+#include <DallasTemperature.h>
+#include <DHT.h>
+
+#define DHTTYPE DHT22   // DHT 22  (AM2302)
+
+/* DECLARE PINS */
+
+// Digital Pins
+const int DS18_PIN = 11; // DS18B20 Temperature (OneWire)
+const int DHT_PIN = 10;  // DHT Temp & Humidity Pin
+
+// Relays
+const int RELAY_0 = A3; // 0_0 PROFET-0 Channel 0 (A3 = 17)
```

Comment:
What does "A3 = 17" mean?

Reply:
I'll probably remove comment altogether. I send relay index numbers right now (0-4) but the previous version had used actual pin numbers where 17 was required instead of A3.